draft-ietf-tls-tls13.md

```diff
@@ -1395,27 +1405,26 @@ the client. If there is overlap in the "supported_group" extension
 but the client did not offer a compatible "key_share" extension,
 then the server will respond with a HelloRetryRequest ({{hello-retry-request}}) message.
 
-If the server selects a PSK, then the PSK will indicate which key
-establishment modes it can be used with (PSK alone or with (EC)DHE) and
-which authentication modes it can be used with (PSK alone or PSK with
-signatures). The server can then select those key establishment and
-authentication parameters to be consistent both with the PSK and the
+If the server selects a PSK, then it MUST only be used with the
+establishment modes indicated by "psk_key_exchange_modes" (PSK alone
```

Comment:
That's intentional, so we don't have duplicate and possible-conflicting indicators.